NeuroID/src/main/java/com/neuroid/tracker/utils/NIDMetaData.kt

```diff
@@ -98,10 +143,17 @@ class NIDMetaData(context: Context) {
         jsonObject.put("isJailBreak", isJailBreak)
         jsonObject.put("isWifiOn", isWifiOn)
         jsonObject.put("isSimulator", isSimulator)
+        getLocation(context)
+        jsonObject.put("gpsCoordinates", gpsCoordinates.toJson())
         return jsonObject
     }
 
     override fun toString(): String = toJson().toString()
 
+    companion object{
+        const val LOCATION_DENIED = "denied"
+        const val LOCATION_UNKNOWN = "unknown"
+        const val LOCATION_AUTHORIZED_ALWAYS = "authorizedAlways"
```

Comment:
Just a small note on the naming. I was initially confused when I gave the app only one time permission to access location but saw LOCATION_AUTHORIZED_ALWAYS in the logs. I'm guessing there's no need to distinguish between one time and long term authorizations.

Reply:
Good call, I don't think android gives us the same level of detail iOS did when it comes to permissions, but maybe we should look to see because we might get a denied next time which would be confusing?

Reply:
will change to "authorized"